.github/workflows/test.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         run: npm run lint:types
 
       - name: Test
-        run: npm test -- --coverage
+        run: npm test -- --coverage --runInBand
```

Comment:
for small amount of tests spawning threads is longer than just run everything in band.